netty/src/test/java/io/grpc/netty/NettyServerBuilderTest.java

```diff
@@ -21,6 +21,8 @@
 
 import io.netty.handler.ssl.SslContext;
 import java.util.concurrent.TimeUnit;
+
```

Comment:
We only add extra lines between static and non-static imports. I'll remove this.

Reply:
I'm sorry for this extra line. I have set the Google Style xml to my IDE.